package.json

```diff
@@ -126,6 +126,7 @@
     "react-lazyload": "^2.3.0",
     "react-mentions": "4.4.2",
     "react-popper": "^2.3.0",
+    "react-resize-panel": "^0.3.5",
```

Comment:
MIT: https://github.com/bjgrosse/react-resize-panel/blob/master/LICENSE

Not the most active repo, but it is an easy to read library and seemed quicker to get started with instead of having to write it all myself